src/Cassandra.IntegrationTests/Core/PreparedStatementsTests.cs

```diff
@@ -606,12 +606,12 @@ public void Bound_Manual_Paging()
             Assert.False(rs.AutoPage);
             Assert.NotNull(rs.PagingState);
             //Dequeue all via Linq
-            var ids = rs.Select(r => r.GetValue<Guid>("id")).ToList();
+            var ids = Enumerable.Select(rs, r => r.GetValue<Guid>("id")).ToList();
             Assert.AreEqual(pageSize, ids.Count);
             //Retrieve the next page
             var rs2 = Session.Execute(ps.Bind().SetAutoPage(false).SetPagingState(rs.PagingState));
             Assert.Null(rs2.PagingState);
-            var ids2 = rs2.Select(r => r.GetValue<Guid>("id")).ToList();
+            var ids2 = Enumerable.Select(rs2, r => r.GetValue<Guid>("id")).ToList();
```

Comment:
Why

Reply:
RowSet now implements both IEnumerable and IAsyncEnumerable which both have a select method. I double checked if it was an issue with my AsyncEnumerable implementation but if you create a new .NET 10 project (that has AsyncEnumerable built-in) and have a class that implements both interface, you'll have this error too.

Reply:
But the driver project only targets netstandard 2.1 so it will never have the built in `AsyncEnumerable` right? Or is this an issue if a client application targeting net10 installs the driver which uses netstandard2.1 and then tries to call `Select` on the `RowSet`?

Reply:
> But the driver project only targets netstandard 2.1 so it will never have the built in AsyncEnumerable right?

Correct, but I provided my own implementation of AsyncEnumerable. It's internal but internals are visible to the test assemblies.

> Or is this an issue if a client application targeting net10 installs the driver which uses netstandard2.1 and then tries to call Select on the RowSet?

Yes this could be an issue.

Reply:
> > Or is this an issue if a client application targeting net10 installs the driver which uses netstandard2.1 and then tries to call Select on the RowSet?

> Yes this could be an issue.

The custom AsyncEnumerable implementation is internal, how would it affect applications installing it?

Reply:
> > But the driver project only targets netstandard 2.1 so it will never have the built in AsyncEnumerable right?
> 
> Correct, but I provided my own implementation of AsyncEnumerable. It's internal but internals are visible to the test assemblies.

But the test assemblies don't target net10 so it shouldn't be an issue now right?